myapp/test/fixtures/labels.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Could you delete `/test` files?
We use `/spec`(rspec) instead of `/test`(minitest)

Reply:
Is it okay to merge after deleting it?

Reply:
yes!